tracsubtickets/web_ui.py

```diff
@@ -70,7 +70,9 @@ def post_process_request(self, req, template, data, content_type):
                 if len(parents) > 0:
                     self._append_parent_links(req, data, ids)
     
-                children = self.get_children(ticket.id)
+                with self.env.db_transaction as db:
```

Comment:
Removing `db` parameter from `get_children` per suggestion below would make this context manager unnecessary.